mungegithub/mungers/inactive-review-handler.go

```diff
@@ -0,0 +1,129 @@
+func (*InactiveReviewHandler) Name() string { return "inactive-review-handler" }
+
+// RequiredFeatures is a slice of 'features' that must be provided
+func (*InactiveReviewHandler) RequiredFeatures() []string {
+	return []string{}
+}
+
+// Initialize will initialize the munger
+func (*InactiveReviewHandler) Initialize(config *github.Config, features *features.Features) error {
+	return nil
+}
+
+// EachLoop is called at the start of every munge loop
+func (*InactiveReviewHandler) EachLoop() error { return nil }
+
+// AddFlags will add any request flags to the cobra `cmd`
+func (*InactiveReviewHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
+
+func (*InactiveReviewHandler) suggestNewReviewer(obj *github.MungeObject, feats *features.Features) string {
+	issue := obj.Issue
+
+	//do not suggest new reviewer if the PR has no author information
+	if issue == nil || issue.PullRequestLinks == nil || issue.User == nil || issue.User.Login == nil {
+		return ""
+	}
+
+	files, ok := obj.ListFiles()
+
+	if !ok || files == nil || len(files) == 0 {
+		return ""
+	}
+
+	potentialOwners, weightSum := getPotentialOwnersHelper(issue.User.Login, feats, files, issue.Number)
+
+	if len(potentialOwners) > 0 && issue.Assignees != nil && len(issue.Assignees) > 0 {
+		oldReviewers := issue.Assignees
+
+		for _, oldReviewer := range oldReviewers {
+			login := *oldReviewer.Login
+
+			for potentialOwner := range potentialOwners {
+				if login == potentialOwner {
+					weightSum -= potentialOwners[login]
+					delete(potentialOwners, login)
+					break
+				}
+			}
+		}
+	}
+
+	if len(potentialOwners) == 0 {
+		return ""
+	}
+
+	newReviewer := selectMultipleOwners(potentialOwners, weightSum, 1)[0]
+	obj.AddAssignee(newReviewer)
+
+	return newReviewer
+}
+
+// Munge is the workhorse encouraging PR author to assign a new reviewer
+// after getting no response from current reviewer for one week
+// The algorithm:
+// (1) find last modification time of the reviewer
+// (2) if the date is one week or longer before today's date, create a comment
+//     encouraging the author to assign a new reviewer and unassign the old reviewer
+// (3) suggest the new reviewer using Blunderbuss algorithm, making sure the old reviewer is not suggested
+func (i *InactiveReviewHandler) Munge(obj *github.MungeObject) {
+	issue := obj.Issue
+	lastModificationTime, ok := findLastModificationTime(obj)
+
+	if !ok {
+		glog.Errorf("failed to obtain last modification time for inactive PR #%v", *issue.Number)
+		return
+	}
+
+	// down to granularity of seconds
+	// number of seconds in one week is 604800
+	if lastModificationTime.Sub(time.Now()).Seconds() < 604800 {
+		return
+	}
+
+	var feats *features.Features
+	requiredFeatures := []string{features.RepoFeatureName, features.AliasesFeature} //based on required features for Blunderbuss
+	feats.Initialize(&github.Config{}, requiredFeatures)                            //TODO: figure out the best way to set config object, and test it
```

Comment:
i was blocked for a day figuring out how should i set up the config and features. i wish to call `getPotentialOwners()` of Blunderbuss. Second argument of the function is `*features.Features`. what is the correct way to obtain the features?

by the way, there is no test file for Blunderbuss.go - i have a hard time testing my codes due to no reference...